pyproject.toml

```diff
@@ -49,10 +49,20 @@ generate    = { script = "tests.generate:main", help = "Generate test cases (do
 test        = { cmd = "pytest --cov src", help = "Run tests" }
 types       = { cmd = "mypy src --ignore-missing-imports", help = "Check types with mypy" }
 format      = { cmd = "black . --exclude tests/output_",   help = "Apply black formatting to source code" }
-clean       = { cmd = "rm -rf .coverage .mypy_cache .pytest_cache dist betterproto.egg-info **/__pycache__ tests/output_*", help = "Clean out generated files from the workspace" }
 docs        = { cmd = "sphinx-build docs docs/build", help = "Build the sphinx docs"}
 bench       = { shell = "asv run master^! && asv run HEAD^! && asv compare master HEAD", help = "Benchmark current commit vs. master branch"}
 
+clean.cmd = """
+rm -rf .asv
```

Comment:
added .asv to the list of files & directories to clean up in the workspace.

Reply:
This change doesn't seem to be valid toml syntax. When running pip install directly from this branch we're getting an `pytoml.core.TomlError: /tmp/pip-req-build-ebkn5z74/pyproject.toml(55, 1):` error.

Reply:
It is valid toml syntax. But pytoml doesn't know that because it's buggy and unmaintained :( 

I guess you're using an older version of pip that depends on it? Let me know if that's not the case.

Reply:
Yes, you're right. 20.0.2 is broken while 20.3.3 is working fine. However I think it would be good to have a bit of backwards compatibility, especially because 20.0.2 is currently shipped with Ubuntu 20.04, so I think multiple users will stumple into this issue. It might also be more consistent when using the same (pytoml compatible) syntax for the clean command, which is also used for docs, bench ...